app/javascript/packs/ajax_operations.js

```diff
@@ -602,31 +601,38 @@ function handle_ajax(event)
           var fbiEntry = fbiApiWord.value
           let text = "<table>"
           for (let x in fbiApiResponseData.items)
-          {
-            if (fbiEntry === 'all')
-            {
-              text += "<tr>";
-              text += "<td>" +
-              fbiApiResponseData.items[x].title + "</td>";
-              text += "<td>" +
-              fbiApiResponseData.items[x].description + "</td>";
-              text += "<td>" +
-              fbiApiResponseData.items[x].details + "</td>";
-              text += "<td>" +
-              fbiApiResponseData.items[x].sex + "</td>";
-              text += "<td>" +
-              fbiApiResponseData.items[x].race_raw + "</td>";
-              text += "<td>" +
-              fbiApiResponseData.items[x].uid + "</td>";
-              text += "<td>" +
-              fbiApiResponseData.items[x].hair_raw + "</td>";
-              text += "<td>" +
-              fbiApiResponseData.items[x].weight + "</td>";
-              text += "<td>" +
-              fbiApiResponseData.items[x].url + "</td>";
-              text += "<td>" +
-              fbiApiResponseData.items[x].person_classification + "</td></tr>";
-            }
+            for (let s in fbiApiResponseData.items[x].subjects)
+              if (fbiApiResponseData.items[x].subjects[s] === 'Kidnappings and Missing Persons' ||
+                  fbiApiResponseData.items[x].subjects[s] === 'ViCAP Missing Persons' )
+              {
+                if (fbiEntry === 'all')
+                {
+
+                     
+                          text += "<tr>";
```

Comment:
I know you didn't create this way of handling the data, but this seems very brittle. At the very least this should be extracted to a separate function.